Hardware/Mainboard/Identification.cs

```diff
@@ -53,6 +53,7 @@ internal class Identification {
         case "LENOVO":
           return Manufacturer.Lenovo;
         case "Micro-Star International":
+        case "Micro-Star International Co., Ltd":

```

Comment:
Given how many of these varations there now are, could you combine the identical names into one and switch based on name.ToUpper()? The other names will need to be converted to as well (Ctrl+Shift+U to uppercase in VS).

Reply:
I think this is better for a upcomming refactoring. The switch case construct is still manageable.

Reply:
Yeah, sure, no problem doing that later, some of the other comments would need to be addressed though.